docs/modules/operation/pages/deep-dive/admin/webui/grafana-dashboard-box.adoc

```diff
@@ -22,78 +24,81 @@ image::webui/startpage/01_grafana-box.png[]
+| The Grafana API key.
+This key is needed for REST calls to work. +
+Refer to the official https://grafana.com/docs/grafana/latest/http_api/[Grafana HTTP documentation] for details on generating an API key.
 | String
-|
+| Empty string
 
 | org.opennms.grafanaBox.tag
-| When a tag is specified only dashboards with this given tag will be displayed.
-When no tag is given all dashboards will be displayed.
+| A tag that specifies which dashboards to display in the Grafana Dashboard Box.
+The tag must be assigned to an existing dashboard for it to be included. +
+When no tag is specified, all dashboards are displayed.
 | String
-|
+| Empty string
 
 | org.opennms.grafanaBox.protocol
-| The protocol for the REST call can also be specified.
+| The protocol for the REST call.
 | String
-| http
+| `http`
 
 | org.opennms.grafanaBox.connectionTimeout
-| Timeout in milliseconds for getting information from the Grafana server.
+| Connection timeout, in milliseconds, for fetching information from the Grafana server.
 | Integer
-| 500
+| `500`
 
 | org.opennms.grafanaBox.soTimeout
-| Socket timeout.
+| Socket timeout, in milliseconds.
 | Integer
-| 500
+| `500`
 
 | org.opennms.grafanaBox.dashboardLimit
-| Maximum number of entries to be displayed (0 for unlimited).
+| The maximum number of entries to display.
+Setting the value to `0` will display an unlimited number of entries.
 | Integer
-| 0
+| `0`
 
 | org.opennms.grafanaBox.link.protocol
-| The protocol used to build each dashboard link URL.
+| The protocol used to build each dashboard URL.
 | String
-| org.opennms.grafanaBox.protocol
+| `org.opennms.grafanaBox.protocol`
 
 | org.opennms.grafanaBox.link.hostname
-| Hostname used to build each dashboard link URL.
+| Hostname used to build each dashboard URL.
 | String
-| org.opennms.grafanaBox.hostname
+| `org.opennms.grafanaBox.hostname`
 
 | org.opennms.grafanaBox.link.port
-| The port used to build each dashboard link URL.
+| The port used to build each dashboard URL.
 | Integer
-| org.opennms.grafanaBox.port
+| `org.opennms.grafanaBox.port`
 
 | org.opennms.grafanaBox.link.basePath
-| The Grafana base path used to build each dashboard link URL.
+| The Grafana base path used to build each dashboard URL.
 | String
-| org.opennms.grafanaBox.basePath
+| `org.opennms.grafanaBox.basePath`
 |===
 
-NOTE: If you have Grafana behind a reverse proxy, you may need to specify the `org.opennms.grafanaBox.link.*` properties.
-The `link.*` properties, if set, are used to build the links that are shown to users, independent of the other properties which are used for internal interaction with the Grafana REST API.
+NOTE: If you have Grafana set up behind a reverse proxy, you may need to define the `org.opennms.grafanaBox.link.*` properties.
+The `link.*` properties, if defined, are used to build the user-facing URLs, independent of other properties that are used for internal interaction with the Grafana REST API.
```

Comment:
This is the corresponding end to the previous `*`
```suggestion
The `link.\*` properties, if defined, are used to build the user-facing URLs, independent of other properties that are used for internal interaction with the Grafana REST API.
```

Reply:
When I tested having both locally, the second escape character was rendered in the compiled docs. Does the same happen for you?